### ISSUE TITLE
Add banner to Edit Group Form Page, Update Gate Topics header 

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#5395]
## Description

Update **Gated Topic(s)**:
- Title should be changed to "Gate Topics".

Make sure the Default Banner is present on both create + edit.

## Project Owner

@zakhap 

## Acceptance Criteria

- Update "Gated Topics" Section Copy, title + paragraph secondly.
- BOTH the Create + Edit Group From Pages should present the Default Banner.
  - The "Edit Group Form Page" needs to have the Default Banner added.

**Comment by CowMuon** [comment]
1 pointer but we're still waiting on Product to remove WIP

**Comment by CowMuon** [comment]
Still need this also to ship @zakhap kudasai

**Comment by jessmart1213** [comment]
HI @zakhap The decisions above are something that design is Responsible, currently feels like I'm being informed. Also I have multiple issues with this tickets, so stating here that Design DOES NOT APPROVE, especially because I see the copy we have currently as being more effective than the above. 

Im open to updating the copy, but Im BLOCKING the above. 

I don't have context for what conversations prompted these changes but will write my feedback here.

---

> Title should be changed to "Gate Topics"

This is fine.

> Paragraph should be updated to:
> Create special permissions for this group! Select Topics below to be gated by this group. Warning: Only group members will be able to participate within the topics selected. Participating includes creating new threads, comments, and reactions. Multiple groups can be given permissions to participate in the same Topic.

Where is the copy above going? This is unclear to me. Im assuming here. 
![Screen Shot 2023-11-07 at 6 41 34 PM](https://github.com/hicommonwealth/commonwealth/assets/133259695/12ae7639-ba53-413e-8a24-c350df93d34f)

- Why are we replacing the current copy "Add topics that only group members who satisfy the requirements above can participate in." if that is the only place on the create and edit groups pages that expands on what gating means?

- Why are we adding another banner ( Im assuming that "informational callout" is the informational banner, this is also unclear ) ? Having 2 banners is cluttered and under-considered. 

- My other issue with this is that this copy is bulky (too long) and redundant to what is in the default banner below, (which has been called an informational banner - these details are important).

- Another issue is that we don't have reactions in our product, so the copy is inaccurate. We have upvoting and commenting, that's it. Reactions are ❤️ emojis. 

> Update Informational Callout at the Bottom of the "Create/Edit Group Form" with the following:
> 
> By default, all community members can participate in all topics. By adding topics to this group (and other groups), participating in the selected topics becomes gated to in-group members only. All threads will still be visible to all community members; only participation is gated. You can always create more groups with different requirements and add any topics to gate, including previously gated topics.

- "By adding topics to this group (and other groups), participating in the selected topics becomes gated to in-group members only." This is sentence is wordy and unclear. 
- "All threads will still be visible to all community members; only participation is gated." This sentence is too general. Our current copy is specific and uses "read" instead of "view"... it uses "create threads, upvote, or comment" instead of "participation". 

----

UX copy is EXTREMELY important and requires specificity in order to create a good user experience.

**Comment by zakhap** [comment]
All unapproved copy was removed from the ticket.
